README.md

```diff
@@ -14,6 +14,9 @@ UCSD DAMS Vagrant Virtual Machine
 
 You can shell into the machine with `vagrant ssh` or `ssh -p 2222 vagrant@localhost`
 
+You can load test record to vagrant by running cURL commands:
```

Comment:
@mcritchlow, the readme has updated to use a damspas example record as you asked.
Tried to squash two commits into one but it failed to push due to conflict between remote and local. So I pushed another commit.